utils/cq/pylintrc

```diff
@@ -9,7 +9,8 @@ good-names=rc,ret,fd,kv,wf,rf
 
 [MESSAGES CONTROL]
 disable=locally-disabled,locally-enabled,star-args,global-statement,bad-option-value,
-        wrong-import-position,modified-iterating-list,duplicate-code,unspecified-encoding
+        wrong-import-position,modified-iterating-list,duplicate-code,unspecified-encoding,
+        too-many-positional-arguments
```

Comment:
Do we have instances of `too-many-positional-arguments` and if we do wouldn't they be real issues?

Reply:
We have _**a lot**_ and most of them are in test code:
https://github.com/daos-stack/daos/actions/runs/10961121721/job/30437303670

It's not saying "you passed too many positional arguments", but rather "the function definition has too many positional arguments".

My intent here was to just stop the bleeding since it's existing

Reply:
I'd missed that distinction, it's a code-smell/refactoring issue rather than a correctness issue.

Another way to silence this would be to set a max value in the config file at the current value we use to prevent further regressions and then ideally start re-working this code but given that it's test code I think what you've done here is fine.